src/Blueprint/Blueprint/Cmdlets/BlueprintDefinition/ExportAzureRMBlueprint.cs

```diff
@@ -0,0 +1,83 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Blueprint.Models;
+using System.IO;
+using System.Management.Automation;
+using Microsoft.Azure.PowerShell.Cmdlets.Blueprint.Properties;
+using static Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants;
+
+namespace Microsoft.Azure.Commands.Blueprint.Cmdlets
+{
+    [Cmdlet("Export", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "BlueprintWithArtifact", SupportsShouldProcess = true, DefaultParameterSetName =
+         ParameterSetNames.ExportBlueprintParameterSet), OutputType(typeof(string))]
```

Comment:
@filizt it doesn't look like we're writing any objects to the output stream, so we should probably change this output type to `bool` and implement the [`-PassThru` parameter](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/cmdlet-best-practices.md#returning-no-output)

Reply:
There is no output produced from this operation. I'll remove the `OutputType(typeof(string))` from the cmdlet.

Reply:
@filizt the cmdlet should still have an output -- the proposed method using `-PassThru` would have `OutputType(typeof(bool))` and would do `WriteObject(true)` at the end of the cmdlet execution if the user provided the `-PassThru` parameter